email_template_qweb/models/mail_template.py

```diff
@@ -40,3 +42,36 @@ def generate_email(self, res_ids, fields=None):
                         result[res_id]['body_html']
                     )
         return multi_mode and result or result[res_ids[0]]
+
+    def _qweb_render_context(self, record):
+        res = {}
+        if record._name == 'mail.message':
+            partner_model = self.env['res.partner']
+            # these variables are usually loaded when the notification is sent
+            # but there are some key values that that are nice to have
+            # if you want to pre-render your template in preview mode.
+            # This is particularly useful if you use the template
+            # from the mail composer, whereas the rendering is done on load.
+            # Also, you don't need to access them from `ctx`.
```

Comment:
I believe all of this should be the docstring of this function, with a little bit of reformulation.

Reply:
the comment is for the call to partner method only. I can cook a docstring tho